docs/cms-adapters/CMS_ADAPTERS.md

```diff
@@ -339,3 +339,58 @@ Make sure to expose them in the bundled application by adding them in the `files
 ```
 
 Also make sure to place them in a folder name `migration-scripts` in your packaged app or to set the `migrationScriptFolder` in your `cms.json`.
+
+### Case of dependencies on libraries
+
+If the libraries that you use are providing migration scripts, you need to aggregate them with your own to make the metadata-checks pass.
+
+You will need to specify in your migration script the list of libraries that you depend on and the current version of the library
```

Comment:
the list of libraries that you depend on => those libraries (depends on + that have migration scripts)